configtx/example_test.go

```diff
@@ -224,12 +224,14 @@ func Example_policies() {
 		panic(err)
 	}
 
-	err = c.RemoveOrdererOrgPolicy("OrdererOrg", configtx.WritersPolicyKey)
+	err = c.UpdatedConfig().Orderer().Organization("OrdererOrg").RemovePolicy(configtx.WritersPolicyKey)
 	if err != nil {
 		panic(err)
 	}
 
-	err = c.SetOrdererOrgPolicy("OrdererOrg", configtx.AdminsPolicyKey, "TestPolicy",
+	err = c.UpdatedConfig().Orderer().Organization("OrdererOrg").SetPolicy(
```

Comment:
Same here; The first half of the chain is the reference that should be maintained.